pvr.dvbviewer/changelog.txt

```diff
@@ -1,3 +1,9 @@
+2.2.0
+[updated] Require DVB Viewer Recording Service 1.30.1 or later
+[updated] implemented PVR_RECORDING.iChannelUid (PVR API v5.0.0)
+[updated] code cleanup
+[fixed] timeshift buffer path: allow escaping from special://userdata
+
 2.1.0
```

Comment:
2.1.1 was released without updating the changelog. Only change was pvr api bump to 5.1.1, no code/other changes. Maybe you want to insert a changelog entry for 2.1.1 anyway?